docs/querying/basics.md

```diff
@@ -83,6 +83,10 @@ metric name that also have the `job` label set to `prometheus` and their
 
     http_requests_total{job="prometheus",group="canary"}
 
+The list of label matchers may include a trailing comma:
```

Comment:
I don't think this justifies a whole paragraph and example. Up at line 77 we should be mentioning that the matchers are separated by commas, and then we can add a mention that trailing commas are okay.

Reply:
Thanks.

Changed this as suggested.